Crash when opening me drive or me chest

[22:03:46] [Server thread/ERROR]: Encountered an unexpected exception
java.lang.NoClassDefFoundError: gregtech/api/GregTech_API
	at appeng.block.storage.BlockDrive.onActivated(BlockDrive.java:56) ~[BlockDrive.class:?]
	at appeng.block.AEBaseTileBlock.func_149727_a(AEBaseTileBlock.java:293) ~[AEBaseTileBlock.class:?]
	at net.minecraft.server.management.ItemInWorldManager.func_73078_a(ItemInWorldManager.java:376) ~[mx.class:?]
	at net.minecraft.network.NetHandlerPlayServer.func_147346_a(NetHandlerPlayServer.java:556) ~[nh.class:?]
	at net.minecraft.network.play.client.C08PacketPlayerBlockPlacement.func_148833_a(SourceFile:60) ~[jo.class:?]
	at net.minecraft.network.play.client.C08PacketPlayerBlockPlacement.func_148833_a(SourceFile:9) ~[jo.class:?]
	at net.minecraft.network.NetworkManager.func_74428_b(NetworkManager.java:212) ~[ej.class:?]
	at net.minecraft.network.NetworkSystem.func_151269_c(NetworkSystem.java:165) ~[nc.class:?]
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:659) ~[MinecraftServer.class:?]
	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:547) ~[MinecraftServer.class:?]
	at net.minecraft.server.integrated.IntegratedServer.func_71217_p(IntegratedServer.java:111) ~[bsx.class:?]
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:427) [MinecraftServer.class:?]
	at net.minecraft.server.MinecraftServer$2.run(MinecraftServer.java:685) [?:?]
Caused by: java.lang.ClassNotFoundException: gregtech.api.GregTech_API
	at net.minecraft.launchwrapper.LaunchClassLoader.findClass(LaunchClassLoader.java:101) ~[launchwrapper-1.12.jar:?]
	at java.lang.ClassLoader.loadClass(Unknown Source) ~[?:1.8.0_421]
	at java.lang.ClassLoader.loadClass(Unknown Source) ~[?:1.8.0_421]
	... 13 more
[22:03:46] [Server thread/ERROR]: This crash report has been saved to: C:\Users\redacted\AppData\Roaming\PrismLauncher\instances\Nuclear Tech Integrated-2.1\minecraft\crash-reports\crash-2024-07-29_22.03.46-server.txt
[22:03:46] [Server thread/INFO]: Stopping server
[22:03:46] [Server thread/INFO]: Saving players
[22:03:46] [Client thread/INFO] [STDOUT]: [net.minecraft.client.Minecraft:func_71377_b:349]: ---- Minecraft Crash Report ----
// Uh... Did I do that?

Time: 29/07/24 22:03
Description: Exception in server tick loop

java.lang.NoClassDefFoundError: gregtech/api/GregTech_API
	at appeng.block.storage.BlockDrive.onActivated(BlockDrive.java:56)
	at appeng.block.AEBaseTileBlock.func_149727_a(AEBaseTileBlock.java:293)
	at net.minecraft.server.management.ItemInWorldManager.func_73078_a(ItemInWorldManager.java:376)
	at net.minecraft.network.NetHandlerPlayServer.func_147346_a(NetHandlerPlayServer.java:556)
	at net.minecraft.network.play.client.C08PacketPlayerBlockPlacement.func_148833_a(SourceFile:60)
	at net.minecraft.network.play.client.C08PacketPlayerBlockPlacement.func_148833_a(SourceFile:9)
	at net.minecraft.network.NetworkManager.func_74428_b(NetworkManager.java:212)
	at net.minecraft.network.NetworkSystem.func_151269_c(NetworkSystem.java:165)
	at net.minecraft.server.MinecraftServer.func_71190_q(MinecraftServer.java:659)
	at net.minecraft.server.MinecraftServer.func_71217_p(MinecraftServer.java:547)
	at net.minecraft.server.integrated.IntegratedServer.func_71217_p(IntegratedServer.java:111)
	at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:427)
	at net.minecraft.server.MinecraftServer$2.run(MinecraftServer.java:685)
Caused by: java.lang.ClassNotFoundException: gregtech.api.GregTech_API
	at net.minecraft.launchwrapper.LaunchClassLoader.findClass(LaunchClassLoader.java:101)
	at java.lang.ClassLoader.loadClass(Unknown Source)
	at java.lang.ClassLoader.loadClass(Unknown Source)
	... 13 more


A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- System Details --
Details:
	Minecraft Version: 1.7.10
	Operating System: Windows 10 (amd64) version 10.0
	Java Version: 1.8.0_421, Oracle Corporation
	Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
	Memory: 762740320 bytes (727 MB) / 3377987584 bytes (3221 MB) up to 7397179392 bytes (7054 MB)
	JVM Flags: 3 total; -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump -Xms512m -Xmx7936m
	AABB Pool Size: 0 (0 bytes; 0 MB) allocated, 0 (0 bytes; 0 MB) used
	IntCache: cache: 0, tcache: 0, allocated: 12, tallocated: 94
	FML: MCP v9.05 FML v7.10.99.99 Minecraft Forge 10.13.4.1614 Angelica 1.0.0-beta2 68 mods loaded, 68 mods active
	States: 'U' = Unloaded 'L' = Loaded 'C' = Constructed 'H' = Pre-initialized 'I' = Initialized 'J' = Post-initialized 'A' = Available 'D' = Disabled 'E' = Errored
	UCHIJAAAA	mcp{9.05} [Minecraft Coder Pack] (minecraft.jar) 
	UCHIJAAAA	FML{7.10.99.99} [Forge Mod Loader] (forge-1.7.10-10.13.4.1614-1.7.10-universal.jar) 
	UCHIJAAAA	Forge{10.13.4.1614} [Minecraft Forge] (forge-1.7.10-10.13.4.1614-1.7.10-universal.jar) 
	UCHIJAAAA	appliedenergistics2-core{rv3-beta-416-GTNH-pre} [Applied Energistics 2 Core] (minecraft.jar) 
	UCHIJAAAA	CodeChickenCore{1.3.6} [CodeChicken Core] (minecraft.jar) 
	UCHIJAAAA	ivtoolkit{1.2.1} [IvToolkit] (minecraft.jar) 
	UCHIJAAAA	NotEnoughItems{2.6.19-GTNH} [NotEnoughItems] (NotEnoughItems-2.6.19-GTNH.jar) 
	UCHIJAAAA	OpenComputers|Core{1.8.5} [OpenComputers (Core)] (minecraft.jar) 
	UCHIJAAAA	debug{1.0} [debug] (denseores-1.6.2.jar) 
	UCHIJAAAA	MysteriumLib{1.1} [§5Mysterium§aLib ] (!MysteriumLib-1.1.jar) 
	UCHIJAAAA	unimixins{0.1.17} [UniMixins] (+unimixins-all-1.7.10-0.1.17.jar) 
	UCHIJAAAA	unimixins-mixin{0.1.17} [UniMixins: Mixin (UniMix)] (+unimixins-all-1.7.10-0.1.17.jar) 
	UCHIJAAAA	unimixins-compat{0.1.17} [UniMixins: Compatibility] (+unimixins-all-1.7.10-0.1.17.jar) 
	UCHIJAAAA	mixingasm{0.3} [UniMixins: Mixingasm] (+unimixins-all-1.7.10-0.1.17.jar) 
	UCHIJAAAA	spongemixins{2.0.1} [UniMixins: SpongeMixins] (+unimixins-all-1.7.10-0.1.17.jar) 
	UCHIJAAAA	mixinbooterlegacy{1.2.1} [UniMixins: MixinBooterLegacy] (+unimixins-all-1.7.10-0.1.17.jar) 
	UCHIJAAAA	gasstation{0.5.1} [UniMixins: GasStation] (+unimixins-all-1.7.10-0.1.17.jar) 
	UCHIJAAAA	gtnhmixins{2.2.0} [UniMixins: GTNHMixins] (+unimixins-all-1.7.10-0.1.17.jar) 
	UCHIJAAAA	mixinextras{0.1.17} [UniMixins: MixinExtras] (+unimixins-all-1.7.10-0.1.17.jar) 
	UCHIJAAAA	appliedenergistics2{rv3-beta-416-GTNH-pre} [Applied Energistics 2] (appliedenergistics2-rv3-beta-416-GTNH-pre.jar) 
	UCHIJAAAA	bdlib{1.9.5.1} [BD Lib] (bdlib-1.9.5.1-mc1.7.10.jar) 
	UCHIJAAAA	ae2stuff{0.8.2-GTNH} [AE2 Stuff] (ae2stuff-0.8.2-GTNH.jar) 
	UCHIJAAAA	gtnhlib{0.3.3} [GTNH Lib] (gtnhlib-0.3.3.jar) 
	UCHIJAAAA	angelica{1.0.0-beta2} [Angelica] (angelica-1.0.0-beta2.jar) 
	UCHIJAAAA	embeddium{1.0.0-beta2} [Embeddium] (angelica-1.0.0-beta2.jar) 
	UCHIJAAAA	notfine{1.0.0-beta2} [NotFine] (angelica-1.0.0-beta2.jar) 
	UCHIJAAAA	mcpatcherforge{1.0.0-alpha} [MCPatcherForge] (angelica-1.0.0-beta2.jar) 
	UCHIJAAAA	AnimationAPI{1.2.4} [AnimationAPI] (AnimationAPI-1.7.10-1.2.4.jar) 
	UCHIJAAAA	AppleCore{3.1.1} [AppleCore] (AppleCore-mc1.7.10-3.1.1.jar) 
	UCHIJAAAA	ArchitectureCraft{1.7.2} [ArchitectureCraft] (ArchitectureCraft-1.7.2-mc1.7.10.jar) 
	UCHIJAAAA	betterbuilderswands{0.8.1} [Better Builder's Wands] (BetterBuildersWands-0.8.1-1.7.10r92+aec06c3.jar) 
	UCHIJAAAA	betterquesting{3.0.328} [Better Questing] (BetterQuesting-3.0.328.jar) 
	UCHIJAAAA	campfirebackport{1.7.10-1.11} [Campfire Backport] (campfirebackport-1.7.10-1.11.jar) 
	UCHIJAAAA	chisel{2.15.1-GTNH} [Chisel] (chisel-2.15.1-GTNH.jar) 
	UCHIJAAAA	controlling{1.7.10-1.0.0.8} [Controlling] (Controlling-1.7.10-1.0.0.8.jar) 
	UCHIJAAAA	MineTweaker3{3.1.0} [MineTweaker 3] (CraftTweaker-1.7.10-3.1.0-legacy.jar) 
	UCHIJAAAA	CustomMainMenu{1.9.2} [Custom Main Menu] (CustomMainMenu-MC1.7.10-1.9.2.jar) 
	UCHIJAAAA	dmod{1.0.3.5} [D-Mod] (DMod-1.7.10-1.0.3.5.jar) 
	UCHIJAAAA	eplus{3.0.2-d} [Enchanting Plus] (EnchantingPlus-1.7.10-3.0.2-d.jar) 
	UCHIJAAAA	etfuturum{2.6.0} [§5§oEt Futurum Requiem] (Et_Futurum_Requiem-2.6.0.jar) 
	UCHIJAAAA	Waila{1.5.10} [Waila] (Waila-1.5.10_1.7.10.jar) 
	UCHIJAAAA	FTBL{1.0.18.2} [FTBLib] (FTBLib-1.7.10-1.0.18.3.jar) 
	UCHIJAAAA	FTBU{1.0.18.2} [FTBUtilities] (FTBUtilities-1.7.10-1.0.18.3.jar) 
	UCHIJAAAA	gravestone{0.7.10.3} [Gravestone] (GraveStone Mod 0.7.10.3.jar) 
	UCHIJAAAA	iChunUtil{4.2.3} [iChunUtil] (iChunUtil-4.2.3.jar) 
	UCHIJAAAA	GraviGun{4.0.0-beta} [GraviGun] (GravityGun-4.0.0-beta.jar) 
	UCHIJAAAA	gregcaves{1.0} [Greg Caves] (gregcaves-1.0.jar) 
	UCHIJAAAA	Harvest{1.7.10-1.0.0-2} [Harvest] (Harvest-1.7.10-1.0.0-2.jar) 
	UCHIJAAAA	hbm{1.0.27 BETA (5036)} [Hbm's Nuclear Tech] (HBM-NTM-.1.0.27_X5036.jar) 
	UCHIJAAAA	inventorytweaks{1.59-dev-152-cf6e263} [Inventory Tweaks] (InventoryTweaks-1.59-dev-152.jar) 
	UCHIJAAAA	ironbackpacks{1.7.10-1.2.20} [Iron Backpacks] (IronBackpacks-1.7.10-1.2.20.jar) 
	UCHIJAAAA	journeymap{5.2.5} [JourneyMap] (journeymap-1.7.10-5.2.5-unlimited.jar) 
	UCHIJAAAA	MTRM{1.0} [MineTweakerRecipeMaker] (MineTweakerRecipeMaker-1.7.10-1.1.1.jar) 
	UCHIJAAAA	MouseTweaks{2.4.4} [Mouse Tweaks] (MouseTweaks-2.4.4-mc1.7.10.jar) 
	UCHIJAAAA	MutantCreatures{1.4.9} [Mutant Creatures] (MutantCreatures-1.7.10-1.4.9.jar) 
	UCHIJAAAA	naturescompass{1.3.1} [Nature's Compass] (NaturesCompass-1.7.10-1.3.1.jar) 
	UCHIJAAAA	neresources{0.1.0.ManuallyBuilt} [Not Enough Resources] (NotEnoughResources-1.7.10-0.1.0-122.jar) 
	UCHIJAAAA	OpenComputers{1.8.5} [OpenComputers] (OpenComputers-MC1.7.10-1.8.5+ad76e2a-universal.jar) 
	UCHIJAAAA	questbook{3.0.0-1.7.10} [Better Questing Quest Book] (questbook-3.0.0-1.7.10.jar) 
	UCHIJAAAA	reccomplex{0.9.7.1.1} [Recurrent Complex] (RecurrentComplex-0.9.7.1.2_Unoffical.jar) 
	UCHIJAAAA	ResourceLoader{1.3} [Resource Loader] (ResourceLoader-MC1.7.10-1.3.jar) 
	UCHIJAAAA	spark{1.10.35} [spark] (spark-1.10.19-forge1710.jar) 
	UCHIJAAAA	bq_standard{3.0.181} [Standard Expansion] (StandardExpansion-3.0.181.jar) 
	UCHIJAAAA	vlzoomer{1.0.0} [VeryLegacyZoomer] (VeryLegacyZoomer-Forge-1.7.10-1.0.0.jar) 
	UCHIJAAAA	WailaHarvestability{1.1.6} [Waila Harvestability] (WailaHarvestability-mc1.7.10-1.1.6.jar) 
	UCHIJAAAA	waystones{1.0.0} [Waystones] (Waystones_Jawsawn_Fork-mc1.7.10-1.0.0.jar) 
	UCHIJAAAA	worldedit{6.1.1} [WorldEdit] (worldedit-forge-mc1.7.10-6.1.1-dist.jar) 
	UCHIJAAAA	denseores{1.0} [Dense Ores] (denseores-1.6.2.jar) 
	GL info: ~~ERROR~~ RuntimeException: No OpenGL context found in the current thread.
	AE2 Version: rv3-beta-416-GTNH-pre for Forge 10.13.4.1614
	List of loaded APIs: 
		* AppleCoreAPI (3.1.0) from AppleCore-mc1.7.10-3.1.1.jar
		* appliedenergistics2|API (rv3) from appliedenergistics2-rv3-beta-416-GTNH-pre.jar
		* BetterQuesting|API (3.2) from BetterQuesting-3.0.328.jar
		* BetterQuesting|API2 (3.1) from BetterQuesting-3.0.328.jar
		* ChiselAPI (0.1.1) from chisel-2.15.1-GTNH.jar
		* ChiselAPI|Carving (0.1.1) from chisel-2.15.1-GTNH.jar
		* ChiselAPI|Rendering (0.1.1) from chisel-2.15.1-GTNH.jar
		* CoFHAPI (1.7.10R1.0.2) from HBM-NTM-.1.0.27_X5036.jar
		* CoFHAPI|energy (1.7.10R1.0.2) from HBM-NTM-.1.0.27_X5036.jar
		* neresources|API (1.0) from NotEnoughResources-1.7.10-0.1.0-122.jar
		* OpenComputersAPI|Component (6.0.0-alpha) from OpenComputers-MC1.7.10-1.8.5+ad76e2a-universal.jar
		* OpenComputersAPI|Core (6.0.0-alpha) from OpenComputers-MC1.7.10-1.8.5+ad76e2a-universal.jar
		* OpenComputersAPI|Driver (6.0.0-alpha) from OpenComputers-MC1.7.10-1.8.5+ad76e2a-universal.jar
		* OpenComputersAPI|Driver|Item (6.0.0-alpha) from OpenComputers-MC1.7.10-1.8.5+ad76e2a-universal.jar
		* OpenComputersAPI|Event (6.0.0-alpha) from OpenComputers-MC1.7.10-1.8.5+ad76e2a-universal.jar
		* OpenComputersAPI|FileSystem (6.0.0-alpha) from OpenComputers-MC1.7.10-1.8.5+ad76e2a-universal.jar
		* OpenComputersAPI|Internal (6.0.0-alpha) from OpenComputers-MC1.7.10-1.8.5+ad76e2a-universal.jar
		* OpenComputersAPI|Machine (6.0.0-alpha) from OpenComputers-MC1.7.10-1.8.5+ad76e2a-universal.jar
		* OpenComputersAPI|Manual (6.0.0-alpha) from OpenComputers-MC1.7.10-1.8.5+ad76e2a-universal.jar
		* OpenComputersAPI|Network (6.0.0-alpha) from OpenComputers-MC1.7.10-1.8.5+ad76e2a-universal.jar
		* OpenComputersAPI|Prefab (6.0.0-alpha) from OpenComputers-MC1.7.10-1.8.5+ad76e2a-universal.jar
		* WailaAPI (1.2) from Waila-1.5.10_1.7.10.jar
	Chisel: Errors like "[FML]: Unable to lookup ..." are NOT the cause of this crash. You can safely ignore these errors. And update forge while you're at it.
	AE2 Integration: IC2:OFF, RotaryCraft:OFF, RC:OFF, BuildCraftCore:OFF, BuildCraftTransport:OFF, BuildCraftBuilder:OFF, RF:ON, RFItem:ON, CoFHWrench:OFF, MFR:OFF, DSU:OFF, FZ:OFF, FMP:OFF, RB:OFF, CLApi:OFF, Waila:ON, InvTweaks:ON, NEI:ON, CraftGuide:OFF, Mekanism:OFF, ImmibisMicroblocks:OFF, BetterStorage:OFF, OpenComputers:ON, PneumaticCraft:OFF, GT:OFF, Chisel:ON, Jabba:OFF, ThaumicTinkerer:OFF
	Mixins in Stacktrace: 
		net.minecraft.network.NetHandlerPlayServer:
			mixins.etfuturum.json:spectator.MixinNetHandlerPlayServer from mod etfuturum
	Profiler Position: N/A (disabled)
	Vec3 Pool Size: 0 (0 bytes; 0 MB) allocated, 0 (0 bytes; 0 MB) used
	Player Count: 1 / 8; [EntityPlayerMP['redacted'/384, l='survival', x=-1306.03, y=54.00, z=2804.53]]
	Type: Integrated Server (map_client.txt)
	Is Modded: Definitely; Client brand changed to 'fml,forge'

--- a/src/main/java/appeng/block/storage/BlockChest.java
+++ b/src/main/java/appeng/block/storage/BlockChest.java
@@ -29,6 +29,7 @@ import appeng.integration.IntegrationRegistry;
 import appeng.integration.IntegrationType;
 import appeng.tile.storage.TileChest;
 import appeng.util.Platform;
+import cpw.mods.fml.common.Loader;
 import cpw.mods.fml.relauncher.Side;
 import cpw.mods.fml.relauncher.SideOnly;
 import gregtech.api.GregTechAPI;

--- a/src/main/java/appeng/block/storage/BlockChest.java
+++ b/src/main/java/appeng/block/storage/BlockChest.java
@@ -57,12 +57,14 @@ public class BlockChest extends AEBaseTileBlock {
                 return true;
             }
 
-            if (IntegrationRegistry.INSTANCE.isEnabled(IntegrationType.GT)
-                    && GTUtility.isStackInList(p.getHeldItem(), GregTechAPI.sWireCutterList)) {
-                if (tg.lockDigitalSingularityCells()) {
-                    p.addChatMessage(PlayerMessages.ChestLocked.get());
+            if (Loader.isModLoaded("dreamcraft")) {
+                if (IntegrationRegistry.INSTANCE.isEnabled(IntegrationType.GT)
+                        && GTUtility.isStackInList(p.getHeldItem(), GregTechAPI.sWireCutterList)) {
+                    if (tg.lockDigitalSingularityCells()) {
+                        p.addChatMessage(PlayerMessages.ChestLocked.get());
+                    }
+                    return true;
                 }
-                return true;
             }
 
             if (side != tg.getUp().ordinal()) {

--- a/src/main/java/appeng/block/storage/BlockDrive.java
+++ b/src/main/java/appeng/block/storage/BlockDrive.java
@@ -54,7 +54,7 @@ public class BlockDrive extends AEBaseTileBlock {
 
         final TileDrive tg = this.getTileEntity(w, x, y, z);
         if (tg != null) {
-            if (Platform.isServer()) {
+            if (Platform.isServer() && Loader.isModLoaded("dreamcraft")) {
                 if (IntegrationRegistry.INSTANCE.isEnabled(IntegrationType.GT)
                         && GTUtility.isStackInList(p.getHeldItem(), GregTechAPI.sWireCutterList)) {
                     if (tg.lockDigitalSingularityCells()) {

--- a/src/main/java/appeng/block/storage/BlockDrive.java
+++ b/src/main/java/appeng/block/storage/BlockDrive.java
@@ -26,6 +26,7 @@ import appeng.integration.IntegrationRegistry;
 import appeng.integration.IntegrationType;
 import appeng.tile.storage.TileDrive;
 import appeng.util.Platform;
+import cpw.mods.fml.common.Loader;
 import cpw.mods.fml.relauncher.Side;
 import cpw.mods.fml.relauncher.SideOnly;
 import gregtech.api.GregTechAPI;

